projects/samples/devices/worlds/console_capabilities.wbt

```diff
@@ -0,0 +1,196 @@
+#VRML_SIM R2020a utf8
+WorldInfo {
+  info [
+    "The MyBot robot, the robot used in the tutorial of Webots"
+  ]
+  title "MyBot"
+}
+Viewpoint {
+  orientation 0.999852653545839 0.011760226481969139 -0.012504729922596871 5.221699933989949
+  position -0.0368013071506027 1.8878786798890779 1.0951532264021957
+}
+TexturedBackground {
+}
+TexturedBackgroundLight {
+}
+RectangleArena {
+}
+Robot {
+  translation 0.337657 -7.99361e-15 -0.334416
+  rotation 0 -1 0 -2.570805307179586
+  children [
```

Comment:
The robot here is not the interesting part, therefore it should be simplified to the extreme.